### DISCUSSION TITLE
isso cannot send username and password

### DISCUSSION THREAD

**Discussion by gflohr** [posativ/isso#788] [Show and tell]
My site uses auth_basic.  When isso "connects" to the site, it gets a 404 but reports no errors.  Subsequent Ajax requests by the browser will then also trigger a 404.

A simple workaround is to turn off auth basic for requests from the server running isso, for example in nginx you would do:

```
satify any;
allow 1.2.3.4:
auth_basic "your auth realm";
auth_basic_user_file "/path/to/users";
```

Since there is a workaround, a fix is not urgent.  But it may make sense to document the problem.  Besides, imho the initial connection from isso to the server should fail or at least spit out a warning.  It is highly unlikely that any other request will succeed, when the intial request for "/" failed.

**Comment by ix5** [comment]
You're welcome to document this. But you're running quite an exotic setup.
